pandas/core/frame.py

```diff
@@ -5960,6 +5960,18 @@ def _construct_result(self, result) -> DataFrame:
         out.index = self.index
         return out
 
+    def __divmod__(self, other):
```

Comment:
shouldn't these be with other arith operators?

Reply:
this is the arithmetic section